### ISSUE TITLE
Cannot find module augmentations

### ISSUE THREAD

**Issue by ttamminen** [xdave/typescript-fsa-redux-thunk#1]
For some reason, when using the library I cannot find the module augmentations that are defined in the library for typescript-fsa:

```
declare module "typescript-fsa" {
  export interface ThunkActionCreators<State, P, S, E>
    extends AsyncActionCreators<P, S, E> {
  }

  export interface ActionCreatorFactory {
    async<S, P, R, E>(prefix?: string, commonMeta?: Meta)
      : ThunkActionCreators<S, P, R, E>;
  }
}
```

This is what I try to do:

```
import actionCreatorFactory from 'typescript-fsa'
import { bindThunkAction } from 'typescript-fsa-redux-thunk'

const actionCreator = actionCreatorFactory()

// cannot find version that would take the State
// only the three generic type version
export const x = actionCreator.async<any, any, any, any>('TEST')
```

**Comment by xdave** [comment]
- What version of Typescript are you using?
- Are you using Visual Studio Code? If so, what version?

**Comment by ttamminen** [comment]
- TypeScript 2.3.4
- Visual Studio Code 1.13.1
- [ts] Supplied parameters do not match any signature of call target.

**Comment by xdave** [comment]
I just tested your code in a new project, and it works.  Could it be your tsconfig.json?

These are the basic options I'm using to make it work.

```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "strict": true,
    "lib": ["esnext"]
  }
}

```

**Comment by ttamminen** [comment]
I think it is some kind of basic mistake from my part. I have tried your tsconfig.json without luck. node_modules folder is deleted and reinstalled. Visual Studio Code has been restarted.

I'll dig a little bit deeper and let you know when I find something.

**Comment by xdave** [comment]
@ttamminen please do, it would suck if this didn't work. lol.  Also, if you're using `yarn`, and you delete `node_modules`, try deleting `yarn.lock`, too.  Same would go for the NPM5 lock file.

**Comment by xdave** [comment]
@ttamminen Another thing, I have the following in my vscode settings:
```
"typescript.tsdk": "./node_modules/typescript/lib",
```

This uses the version of typescript that is installed in the current project rather than the one bundled with visual studio code.

Does adding this setting and restarting vscode help?

**Comment by ttamminen** [comment]
I think I was able to solve the issue!

It seems that one of the @types packages didn't install correctly with the existing npm version. I am using Windows and got errors related to permissions (EPERM), renaming stuff, etc. Even when running as an admin I got these errors.

I updated to latest npm version, and the same problem continued. Switched to Yarn, installed everything successfully. 

After restarting Visual Studio Code, all module augmentations were visible!

Thanks @xdave for troubleshooting help!
